doc/examples/pulsating-sphere-displacement.py

```diff
@@ -0,0 +1,72 @@
+"""
+Particle displacement of a pulsating sphere.
+"""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation, patches
+
+# Pulsating sphere
+x0 = [0, 0, 0]  # position
+a = 0.25  # radius
+d_peak = 0.05  # peak value of the surface displacement
+f = 1000  # frequency
+omega = 2 * np.pi * f  # angular frequency
+ka = sfs.util.wavenumber(omega) * a
+
+# Temporal sampling for animation
+fs = f * 10  # sampling frequency
+L = int(np.round(fs / f))  # number of frames corresponding to one period
+t = np.arange(L) / fs  # time
+
+# Uniform grid
+N = 120
+xmin, xmax = -1, 1
+ymin, ymax = -1, 1
+x = np.tile(np.linspace(xmin, xmax, N), N)
+y = np.repeat(np.linspace(ymin, ymax, N), N)
+z = 0
+grid = sfs.util.XyzComponents([x, y, z])
+
+# Particle displacement
+d = sfs.mono.source.pulsating_sphere_displacement(omega, x0, a, d_peak, grid)
+
+# Animation
+fig = plt.figure(figsize=(8, 8))
+ax = plt.axes(xlim=(-xmax, xmax), ylim=(-ymax, ymax))
+circ_patch = ax.add_patch(patches.Circle(x0[:2],
+                                         radius=a,
+                                         facecolor='RoyalBlue',
+                                         linestyle='None'))
+line, = ax.plot(grid[0], grid[1],
+                marker='o',
+                markersize=1.5,
+                markerfacecolor='gray',
+                markeredgecolor='None',
+                linestyle='None',
+                alpha=0.75)
```

Comment:
I used `pyplot.plot` because it returns a `line` object which makes it easier to update within `animation.FuncAnimation()`. If I get it right, `sfs.plot.particles` isn't returning anything.

Reply:
Feel free to change `sfs.plot.particles()` to return whatever makes sense!